packages/bundler/package.json

```diff
@@ -1,24 +1,24 @@
 {
-  "name": "@cadl-lang/bundler",
+  "name": "@typespec/bundler",
   "version": "0.1.0",
   "author": "Microsoft Corporation",
-  "description": "Package to bundle a cadl library.",
-  "homepage": "https://github.com/Azure/cadl",
-  "readme": "https://github.com/Azure/cadl/blob/master/README.md",
+  "description": "Package to bundle a typespec library.",
+  "homepage": "https://github.com/Azure/typespec",
```

Comment:
Want to update this to `typespec.io` as part of this changeset?

Reply:
Let's track this as a manual follow up along with others.

Reply:
in PR